reactioncommerce/reaction
# Prevent archived products from showing in search

Fixes [#1730](https://github.com/reactioncommerce/reaction/issues/1730) 

Expected behavior: Once a product is archived, it should no longer appear in product search.

**Steps to test:**
1. Create a product and publish. Confirm that it shows when you search for the product.
2. Archive the product, then confirm that the product does not show again when searched for.
3. Add other products and confirm that search still works as before.
